docs/index.html

```diff
@@ -55,7 +55,7 @@
 
   <body>
     <div id="app">Loading ...</div>
-    <script src="//cdn.jsdelivr.net/npm/docsify-plugin-carbon@1/index.js"></script>
+    <script src="//cdn.jsdelivr.net/npm/docsify-plugin-carbon@1"></script>
```

Comment:
Do you need to update index.html?

https://github.com/docsifyjs/docsify/blob/533193fd2d27f7edae051ea4de80514c09545248/index.html#L26

Reply:
Yep. :)

Done.